std/typecons.d

```diff
@@ -3875,14 +3875,118 @@ unittest
 }
 
 /**
+ * Determines if the $(D Source) type satisfies all interface requirements of
+ * $(D Targets)
```

Comment:
Unfinished sentence. Generally this needs to be fleshed into real documentation for acceptance.